src/editors/containers/ProblemEditor/data/ReactStateOLXParser.js

```diff
@@ -134,7 +139,8 @@ class ReactStateOLXParser {
   }
 
   addQuestion() {
-    const { question } = this.editorObject || this.problemState;
```

Comment:
why remove this object dereferencing?  your change can still produce null as a result?

Reply:
In a later commit, I change it. I also just realized that question should always be defined so I can change it to
```
const { question } = this.editorObject
```